server/polar/transaction/service/refund.py

```diff
@@ -116,6 +123,42 @@ async def create(self, session: AsyncSession, refund: Refund) -> Transaction:
             payment_transaction=payment_transaction,
             refund_amount=settlement_amount - settlement_tax_amount,
         )
+
+        customer = payment_transaction.payment_customer
+        organization = payment_transaction.payment_organization
+        order = payment_transaction.order
+        if customer is not None and organization is not None:
+            metadata: BalanceRefundMetadata = {
+                "transaction_id": str(refund_transaction.id),
+                "refund_id": str(refund.id),
+                "amount": refund_transaction.amount,
+                "currency": refund_transaction.currency,
+            }
+            if order is not None:
+                metadata["order_id"] = str(order.id)
+                metadata["product_id"] = str(order.product_id)
+                if order.subscription_id is not None:
+                    metadata["subscription_id"] = str(order.subscription_id)
+                refund_total = refund.amount + refund.tax_amount
+                order_total = order.total_amount
+                total_refunded = order.refunded_amount + order.refunded_tax_amount
+                metadata["is_partial"] = refund_total < order_total
+                metadata["completes_refund"] = total_refunded >= order_total
```

Comment:
Should this only be looking at `refunded_amount` and not include `refunded_tax_amount`?